cmd/flags.go

```diff
@@ -328,6 +351,21 @@ func ValidateNoArgs(ctx *cli.Context) error {
 	return nil
 }
 
+func ValidateNetworkFlags(ctx *cli.Context) error {
+
```

Comment:
Unnecessary leading newline (the CI fails because of this).

Reply:
resolved this in the next commit